hooks/tk-multi-publish2/maya/collector_integ.py

```diff
@@ -0,0 +1,42 @@
+import maya.cmds as cmds
+import sgtk
+from sgtk import TankError
+from sgtk.templatekey import SequenceKey
+
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class MayaSessionCollector(HookBaseClass):
+    """
+    Inherits from maya collector.
+    Adds maya session based attributes to item.properties.fields
+    """
+    def _resolve_item_fields(self, settings, item):
+        """
+        Helper method used to get fields that might not normally be defined in the context.
+        Intended to be overridden by DCC-specific subclasses.
+        """
+        # Now run the parent resolve method
+        fields = super(MayaSessionCollector, self)._resolve_item_fields(settings, item)
+        publisher = self.parent
+        if item.properties.is_sequence:
```

Comment:
Is this condition working? `maya.session` item has `is_sequence` hard-coded to `False`.

Reply:
yes the condition is working properly